crates/node/p2p/src/discv5/driver.rs

```diff
@@ -125,25 +155,29 @@ impl Discv5Driver {
                 Err(e) => debug!("Failed to add ENR to discv5 table: {:?}", e),
             }
         }
-        info!("Added {} ENRs to discv5 | {} in bootstore", count, self.store.len());
 
         // Merge the bootnodes into the bootstore.
         self.store.merge(boot_enrs);
+        info!("Added {} ENRs to discv5 | {} in bootstore", count, self.store.len());
```

Comment:
can be downgraded to debug log level. and pls add so we can filter when debugging. we also tend to use the k=v format for values. see lighthouse or reth for reference.
```suggestion
        debug!(target: "p2p::discv5",
            new=%count,
            total=%self.store.len(),
            "Added new ENRs to discv5 bootstore"
        );
```
then we debug like,
`$ RUST_LOG=p2p::discv5=trace <kona-cmd>`